### DISCUSSION TITLE
Follow-up to question #326

### DISCUSSION THREAD

**Discussion by pv2-0** [eclipse/mosaic#339] [Q&A] [answered]
I am having the same issue that was described in question #326. I pulled all the updates a few days ago. I checked, and in my AbstractSumoAmbassador.java file, line #873 is properly updated.

However, I tried running my scenario again, and nothing changed - I am still getting the same error. Do I need to recompile the .java file in some way? Or is the mosaic-sumo-23.0-SNAPSHOT.jar file in the lib folder automatically updated?

Thanks

**Comment by schwepmo** [comment, marked as answer]
After you pull the changes you should always call `mvn clean install` in the root directory. If you're using the `MosaicStarter` to start your scenarios from within your IDE afterwards everything should work. Otherwise, if you're using the bundled MOSAIC version you would need to re-extract the files generated in the `bundle/target/eclipse-mosaic<...>.<tar/zip>`
